tasks/install_agent_shell.sh

```diff
@@ -0,0 +1,690 @@
+#!/usr/bin/env bash
+
+set -e
+
+#####
+# Install openvox-agent as a task
+#
+
+# Error if non-root
+if [ `id -u` -ne 0 ]; then
```

Comment:
we don't have shellcheck in our pipeline right now. Could you/did you validate the file with shellcheck and check it for potential styleguide validations? I think it e.g. prefers `$()` over ``

Reply:
I think your right.

Reply:
done